src/apps/seelenweg/components/WithContextMenu.tsx

```diff
@@ -22,19 +23,29 @@ export function WithContextMenu({ children, items }: Props) {
     <Dropdown
       placement="topLeft"
       open={openContextMenu}
-      onOpenChange={setOpenContextMenu}
+      onOpenChange={(isOpen) => {
+        setOpenContextMenu(isOpen);
+        if (onOpenChange)
+          onOpenChange(isOpen);
+      }}
       trigger={['contextMenu']}
       dropdownRender={() => (
-        <BackgroundByLayersV2
-          className="weg-context-menu-container"
-          prefix="menu"
-        >
-          <Menu
-            className="weg-context-menu"
-            onMouseMoveCapture={(e) => e.stopPropagation()}
-            items={items}
-          />
-        </BackgroundByLayersV2>
+        items.length != 0 ?

```

Comment:
![image](https://github.com/user-attachments/assets/c4d349f5-3b54-4a99-8b1a-b42da10449fa)

![image](https://github.com/user-attachments/assets/14d4426c-a99c-4800-b830-bf0c66b50790)

If this is important for you, then I remove from there, where I use this: 
![image](https://github.com/user-attachments/assets/7e15aad0-7266-48ee-b475-ef9da7215719)

This was in the description, I used this way, because this should be filled with items, but this pr is not about to change functions: 
![image](https://github.com/user-attachments/assets/314ea35d-50ef-47a6-9310-52ff2ec7218a)

if I should not do this, I remove the handling from ContextMenu and add here to e.stopPropagation()